icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -22,27 +60,39 @@ class RepositoryConfig:
         """
         ...
 
+    @property
+    def inline_chunk_threshold_bytes(self) -> int: ...
+    @property
+    def unsafe_overwrite_refs(self) -> bool: ...
+    @property
+    def virtual_chunk_containers(self) -> dict[str, VirtualChunkContainer]: ...
+    def set_virtual_chunk_container(self, cont: VirtualChunkContainer) -> None: ...
+    def clear_virtual_chunk_containers(self) -> None: ...
+
 class PyRepository:
     @classmethod
     def create(
         cls,
         storage: Storage,
         *,
         config: RepositoryConfig | None = None,
+        virtual_chunk_containers: dict[str, Credentials] | None = None,
```

Comment:
should these be named `virtual_chunk_containers` or `virtual_chunk_credentials`?

Reply:
good catch ... fixed